### ISSUE TITLE
[Help]: [Makefile:49: envi_slc_decode] Error 1, [Makefile:49: install-preproc] Error 2

### ISSUE THREAD

**Issue by dr14nium** [gmtsar/gmtsar#588]
**Describe the problem you met**
Hello!
I'm just starting to use GMTSAR and encounter problems during the installation. Previously I had installed GMT, then I continued to install GMTSAR. When I was at the https://github.com/gmtsar/gmtsar/wiki/GMTSAR-Wiki-Page stage, to be more specific, when I did the "make" & "make install" commands, I always encountered an error like the image below. I've tried to reinstall my Linux and start from 0 but the same error always appears.

**OS and software version**
If applicable, what operating system and software version you are using?
OS: Ubuntu 22.04.1 LTS
GMTSAR version: 6.2

**Reproduce the problem**
If applicable, how to reproduce the problem from scratch?

**Log file**

**Screenshot**
![image](https://user-images.githubusercontent.com/74701645/205477568-8fba9e3a-c959-4fef-84b9-3fc950659601.png)

[Please note that if no new responses are made over six months from users, the issue will get closed]

[Also if this is a general question that's not particlarly related to GMTSAR, such as "how to correct atmospheric noise" or "how to intepret my interferogram", consider using the discussions page]

**Comment by brioglade** [comment]
No. I have the same problem. @Xiaohua-Eric-Xu Could you please fix this out?

**Comment by Xiaohua-Eric-Xu** [comment]
> No. I have the same problem https://bsd.to/N6oy. @Xiaohua-Eric-Xu Could you please fix this out?

The cause is because many of the codes were written when the compilers were not as picky. A real way to fix this is to go over every early code and make sure all functions have unique names which will take a very long time to do. Alternatively one could also use a compiler that allows multiple definitions (less picky) of functions or do as suggested above. If you really don't want to bother with installation, use homebrew or try our virtual machine here: http://topex.ucsd.edu/gmtsar/tar/Ubuntu21.04_gmt6.1_gmtsar6.2.ova

**Comment by Xiaohua-Eric-Xu** [comment]
BTW, we cannot chose what default compiler every system uses. This was not a problem for Ubuntu18 or earlier. It started to show up for Ubuntu 20 or later since they used newer (or picker) compilers.

**Comment by brioglade** [comment]
OK. Thank you.
